boosters/node-beginner/tests/02_exercise.test.js

```diff
@@ -1,13 +1,13 @@
 const request = require("supertest");
-const app = require("../src/app");
-const userSetup = require("./config/setup")("Users");
-const repoSetup = require("./config/setup")("Repos");
+const app = require("../src/server");
+const userSetup = require("./config/setup");
```

Comment:
Since `setup` is refactored, we would only need to use one setup variable.

Reply:
Nice catch thanks